packages/fev4-fpm-writer/templates/page/1.94/ext/View.xml

```diff
@@ -0,0 +1,12 @@
+<mvc:View xmlns:core="sap.ui.core" xmlns:mvc="sap.ui.core.mvc" xmlns="sap.m"
+	xmlns:html="http://www.w3.org/1999/xhtml" controllerName="<%= id %>.<%= name %>">
+	
+	<App>
```

Comment:
Why do you wrap the Page in an App ? This is not needed nor is it natural

Reply:
Hmm ... I think this was taken from https://ui5.sap.com/test-resources/sap/fe/core/fpmExplorer/index.html#/customElements/customElementsOverview/customPageContent. If I knew the author of it ...

Reply:
![image](https://user-images.githubusercontent.com/13470207/139824871-99948e34-b4aa-4675-82ba-0f65695a3bcf.png)
Doesn't seem like it :)

Reply:
At least we use this pattern for the V2 custom pages all the time and it works, but we can try a simplification for V4